### ISSUE TITLE
How to use this plugin ?

### ISSUE THREAD

**Issue by yet-another-average-joe** [5axes/TabAntiWarping#2]
Hello,

I'm just testing TabAntiWarping for the first time.
I get weird results, maybe I don't use it as it should be...

I get this : a "yellow" grid pattern.

![Capture1](https://user-images.githubusercontent.com/43514870/90196999-a7b65500-ddcd-11ea-925f-f7bb6790251a.PNG)

The yellow pattern looking like an infill, I changed the infill density to 100% using the "per model settings". I got this : 2 overlapping objects : the result is not printable.

![Capture2](https://user-images.githubusercontent.com/43514870/90197189-32974f80-ddce-11ea-9ad4-ae7b1b816ea3.PNG)

How do I get a cyan tab, part of the "adhesion gcode", as shown on the plugin homepage ?

Tested with Cura 4.6.2

**Comment by cachrisman** [comment]
I just tried using this for the first time and I changed the `Support Density` setting to `100%` to get Cura to make solid tabs

![image](https://user-images.githubusercontent.com/6521666/90252157-5050cd00-de3f-11ea-8a2c-789d461193c4.png)

**Comment by 5axes** [comment]
works fine for me .. Perhap's an issue with multi-extruder machine ?
![Tab](https://user-images.githubusercontent.com/11015345/91030231-6c066100-e5ff-11ea-8b6c-ff673fbb0a17.JPG)

**Comment by 5axes** [comment]
To get the right result I'm using the option Enable Support Brim in the Support settings section
![support](https://user-images.githubusercontent.com/11015345/91038259-c5c05880-e60a-11ea-8eb1-d41298089cd5.JPG)

**Comment by yet-another-average-joe** [comment]
Thanks !
Now it works fine.

In think I understand how it works : TabAntriWarping generates one layer support objects.

You should add "Enable Support Brim" in the "How to use" instructions !

**Comment by 5axes** [comment]
https://github.com/5axes/TabAntiWarping/wiki/How-to-use-it

**Comment by yet-another-average-joe** [comment]
Shouldn't it be also on the home page https://github.com/5axes/TabAntiWarping ?

**Comment by dapostol73** [comment]
By default Cura has "Enable Support Brim" set to False for the regular FDM prints definition, seems some definitions set this to true by default.  Is there a way we could set "Enable Support Brim" to True in the python script?  Without so it works by default.

**Comment by 5axes** [comment]
Thank you for this idea. 
But it's not so obvious, as support_brim_enable is settable per extruder , how to manage this ? fix the value for every extruder ?

**Comment by dapostol73** [comment]
My initial thought would be that if there is only 1 extruder, it does it automatically with a notification.  If there is more than one, it would prompt the user with a dialog box that would ask them which extruder they would like it applied, extruder0, extruder1, etc. with an option for All for the first tab they place, then all others would follow the same logic.

**Comment by fvgennip** [comment]
Hi, I just installed the plugin on my mac (Big Sur, v11.4), using Cura 4.9.1. I do see the option to select a Tab, I'm also able to change settings, but I cannot add a Tab to my design. After selecting an object in Cura, I can press "I" or select the option from the menu bar in the left, but clicking on my model doesn't do anything (except unselecting the selected object...).
Any hints? Tips?

Thanks.

**Comment by 5axes** [comment]
You can find on YouTube some video concerning this plugin, like this one :  https://youtu.be/D45eIhoWSyg

**Comment by fvgennip** [comment]
> You can find on YouTube some video concerning this plugin, like this one : https://youtu.be/D45eIhoWSyg

Thanks! I got it working now!

**Comment by MattLogue** [comment]
Nope. Does not work. Takes 10 minutes to slice this. Maybe I should of tried on something not a 30MB lithophane... any how I can't get it to print the tabs, just shades that area. That's after finding out the secret requirement of using supports and brim, only in this issues, not in instructions.
